chalicelib/checks/wrangler_checks.py

```diff
@@ -243,6 +243,18 @@ def biorxiv_is_now_published(connection, **kwargs):
             # we have possible article(s) - populate check_result
             fndcnt += 1
             fulloutput['biorxivs2check'][buuid] = ['PMID:' + id for id in ids]
+            # look for GEO datasets
+            for id_ in ids:
+                result = requests.get('https://eutils.ncbi.nlm.nih.gov/entrez/eutils/'
```

Comment:
Should this be in a try-except block as it's ancillary and not sure check should error if the get doesn't work?

Reply:
I  could do that - should I do the same for all gets? e.g. line 220 `res = requests.get(title_query)`?

Reply:
Yes - try-except is not the only option - just need some way to be sure that result.text is something you can actually work with